README.md

```diff
@@ -77,7 +79,7 @@ source = records.sources.dataframe(df=df)
 target = records.targets.table(schema_name='myschema',
                                table_name='mytable',
                                db_engine=db_engine)
-results = records.move(source, target)
+results = move(source, target)
```

Comment:
Hmm... This is the only one I'm wondering about... Does it maybe make sense to keep `move` as part of the `records` object, along with the factory functions? Right now, `move` is a simple function that gets everything from sources and targets-- but it's very plausible that it will eventually take other configuration that may make sense to live in the session. A few examples off the top of my head: hints about how to weight different strategies when we have more negotiation; temporary storage configuration for intermediate steps.

Of course, in those cases, `move` could also take a session parameter-- so could probably make a stylistic argument either way.

What do you think?

Reply:
Makes total sense to me - done.